src/utils/sortObjectKeys.ts

```diff
@@ -1,28 +1,33 @@
 import { Settings, SortOrder } from "../types/settings";
 
-function sortObjectKeys(obj: any, sortOrder: SortOrder): any {
-  if (sortOrder === SortOrder.NONE) return obj;
+const sortHandlers = {
+  [SortOrder.ASC]: (object: Record<string, any>) =>
+    Object.fromEntries(
+      Object.entries(object).sort(([a], [b]) => a.localeCompare(b))
+    ),
 
-  if (Array.isArray(obj)) {
-    return obj.map((item) => sortObjectKeys(item, sortOrder));
-  } else if (obj !== null && typeof obj === "object") {
-    // Convertimos el objeto a un array de [clave, valor] y aplicamos recursión en los valores
-    const entries = Object.entries(obj).map(([key, value]) => [
-      key,
-      sortObjectKeys(value, sortOrder),
-    ]);
+  [SortOrder.DESC]: (object: Record<string, any>) =>
```

Comment:
here is a reusable GenericObject type that you can use here
https://github.com/heliomar-pena/i18n-populator/blob/master/src/types/shared.d.ts